docs/design-ux/sharing.md

```diff
@@ -0,0 +1,104 @@
+  teams with subteams that have at least as much access. The designs we consider are:
+
+1. Root sharing
+2. Folder sharing
+3. Policy Sharing
+
+## 1. Root Sharing
+
+### UX
+
+Root sharing is similar to 1Password's vaults or Slack's workspaces. Under root sharing, users can create a new root and
+share that root with any set of users. Those users will have access to all files under that root. This means each set of
+sharees requires its own root. Paths for files under the new root begin with the root's name. The root's name is unique
+with usernames - no root name can be the same as any other root name or any username (similar to GitHub's rule that no
+org have be the same name as any other org or any user).
+
+### Technical Implementation
+
+Under this model, roots need only user access keys and non-root files need only folder access keys. This is a
+simplification of what we've already implemented.
+
+### For Individuals
+
+This model works poorly for individuals because they need to organize their documents according to what is shared with
+who. If I want to share my reading list with Parth, I need to create a vault shared with Parth and move my reading list
+into it. If I then want to share my notes on Malcom Gladwell's _Outliers_ with Parth and Raayan, I need to create a
+second vault shared with both of them and move my notes into it. Those notes would no longer live alongside all my other
+notes on books. A solution for this would be a symlink system, so that my notes can live in a shared vault but I can
+pretend they also live in a certain folder in my personal lockbook. The sharing workflow would create a vault for me and
+the sharee, move the file into it, then create a symlink for the file where it was originally. This would complicate
+linking logic because files no longer have a single unique path.
+
+### For Teams
+
+This model works well for teams because they can make a single team vault. As a member of the team, all my personal
+thoughts would live in my personal root whose path starts with my username and all my team thoughts would live in my
+team's root. I would be able to tell at a glance which files are shared with who and I would not be worried about
+accidentally sharing a personal thought with the team.
+
+### For Enterprises
+
+This model works okay for enterprises. In an enterprise, I would be part of multiple subteams e.g. my team, my
+department, and the whole organization. Because each set of sharees needs its own root, I would have one root for each
+team I am on. This makes it easy to share a file and know who it's shared with but difficult to organize across teams.
+For example if my team has technical documents that should be shared with an architecture council and cost estimation
+documents that should be shared with the budget team, we wouldn't be able to organize those in the same folder tree.
+
+## 2. Folder Sharing
+
```

Comment:
I think this doesn't work well for absolute links. Under vault-sharing every file has an absolute link and you could link a `lb://babyloncove/readinglist.md` in slack. In folder sharing, either the links are subjective, or they leak information about *my* folder structure, right?

Reply:
yeah one way or another it's a loss for links

Reply:
well, there's always permalinks e.g. `lb://123e4567-e89b-12d3-a456-426614174000`